src/mumble/Log.ui

```diff
@@ -176,7 +176,7 @@
      <property name="title">
       <string>Chat Log</string>
      </property>
-     <layout class="QGridLayout" name="gridLayout">
+     <layout class="QGridLayout" name="_2">
```

Comment:
Nope. Warning emitted because Qt creates "gridLayout" as the default name of the first grid layout in that file which collides with the old value. I just hit reset in the designed and it reset it to "_2". Not sure if I could completely remove it and didn't consider it worth questioning.